mods/ctf/ctf_bandages/init.lua

```diff
@@ -1,9 +1,9 @@
 --Inspired from Andrey's bandages mod
 
-local healing_limit = 15
+heal_percent = 0.75 --Percentage of total HP to be healed
```

Comment:
Apologies if I'm being too pedantic. But it's a bad idea to expose a mod's internal variable as a global. I suggest adding this to the mod's namespace, as I earlier suggested.

```suggestion
ctf_bandages = {}
ctf_bandages.heal_percent = 0.75
```